### PR TITLE
[Doc] add http proxy example

### PR DESCRIPTION
**Pull request by kaisecheng**
Document HTTP proxy usage for GeoIP service endpoint
This feature has been backported to Logstash 7.17.2, 8.1.0

Fixed: #200
Related: https://github.com/elastic/logstash/pull/13410

### CODE REVIEW

File: docs/index.asciidoc [hunk 1]
```diff
@@ -75,11 +75,15 @@ TIP: When possible, allow Logstash to access the internet to download databases
 [id="plugins-{type}s-{plugin}-manage_update"]
 ==== Manage your own database updates
 
-**Use a proxy endpoint**
+**Use a HTTP proxy**
```
Comment (karenzone) [inline docs/index.asciidoc:78]:
```suggestion
**Use an HTTP proxy**
```

File: docs/index.asciidoc [hunk 2]
```diff
@@ -75,11 +75,15 @@ TIP: When possible, allow Logstash to access the internet to download databases
...
 
 If you can't connect directly to the Elastic GeoIP endpoint, consider setting up
-a secure proxy. You can then specify the proxy endpoint URL in the
-`xpack.geoip.download.endpoint` setting in `logstash.yml` file.
+a HTTP proxy server. You can then specify the proxy with `http_proxy` environment variable.
```
Comment (karenzone) [inline docs/index.asciidoc:81]:
```suggestion
an HTTP proxy server. You can then specify the proxy with `http_proxy` environment variable.
```

**Review by karenzone — APPROVED** [review]
Added suggestions for very minor correction. Otherwise, LGTM. Thanks for this clarification and for adding the example.

**Comment by kaisecheng** [comment]
CI red due to https://github.com/elastic/logstash/issues/13890

**Pull request merged**